test/recipes/75-test_quicapi_data/ssltraceref.txt

```diff
@@ -13,12 +13,14 @@ Header:
         {0x13, 0x01} TLS_AES_128_GCM_SHA256
       compression_methods (len=1)
         No Compression (0x00)
-      extensions, length = 209
+      extensions, length = 214
         extension_type=UNKNOWN(57), length=49
           0000 - 0c 00 0f 00 01 04 80 00-75 30 03 02 44 b0 0e   ........u0..D..
           000f - 01 02 04 04 80 0c 00 00-05 04 80 08 00 00 06   ...............
           001e - 04 80 08 00 00 07 04 80-08 00 00 08 02 40 64   .............@d
           002d - 09 02 40 64                                    ..@d
+        extension_type=renegotiate(65281), length=1
+            <EMPTY>
```

Comment:
This indicates that the renegotiation extension is sent even on TLS-1.3 only client hello.